src/modules/dashboard/components/ExpenditurePage/Stages/StreamingStages/StreamingStagesLocked/StreamingStagesLocked.tsx

```diff
@@ -288,13 +306,14 @@ const StreamingStagesLocked = ({
                   const token = colony?.tokens?.find(
                     (tokenItem) => tokenItem.address === availableItem.token,
                   );
+                  if (!token) {
+                    return null;
+                  }
 
                   return (
                     <span
                       className={classNames(styles.value, {
-                        [styles.marginBottom]:
-                          availableToClaim.length > 1 &&
-                          index !== availableToClaim.length - 1,
+                        [styles.marginTop]: index > 0,
```

Comment:
Similarly to my question in #3888, wouldn't it make sense for that to be done in CSS?

Reply:
I agree, I think you can use css selectors for that.